diff --git a/node_modules/@consento/sync-randombytes/sync-randombytes.podspec b/node_modules/@consento/sync-randombytes/sync-randombytes.podspec
index 8ccee8d..70eb4e2 100644
--- a/node_modules/@consento/sync-randombytes/sync-randombytes.podspec
+++ b/node_modules/@consento/sync-randombytes/sync-randombytes.podspec
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/consento-org/sync-randombytes.git", :tag => "v#{s.version}" }
   s.source_files  = ["ios/**/*.{h,m}"]
 
-  s.dependency 'React'
+  s.dependency "React-Core"
 end
