diff --git a/node_modules/@react-native-community/push-notification-ios/RNCPushNotificationIOS.podspec b/node_modules/@react-native-community/push-notification-ios/RNCPushNotificationIOS.podspec
index ea0f1f7..0c8407d 100644
--- a/node_modules/@react-native-community/push-notification-ios/RNCPushNotificationIOS.podspec
+++ b/node_modules/@react-native-community/push-notification-ios/RNCPushNotificationIOS.podspec
@@ -18,6 +18,6 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, "9.0"
 
-  s.dependency "React"
+  s.dependency "React-Core"
 
 end
