diff --git a/node_modules/react-native-permissions/RNPermissions.podspec b/node_modules/react-native-permissions/RNPermissions.podspec
index 4a668cf..3a35143 100644
--- a/node_modules/react-native-permissions/RNPermissions.podspec
+++ b/node_modules/react-native-permissions/RNPermissions.podspec
@@ -3,7 +3,7 @@ package = JSON.parse(File.read('./package.json'))
 
 Pod::Spec.new do |s|
   s.name                      = "RNPermissions"
-  s.dependency                  "React"
+  s.dependency "React-Core"
 
   s.version                   = package["version"]
   s.license                   = package["license"]
